samples/ComponentExamples/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "component-examples",
+  "version": "0.0.1",
+  "private": true,
+  "description": "This is a sample application to show how the ACS Chat Web SDK can be used to build a group chat experience. The client-side application is a React based user interface which uses React Hooks for handling complex state while leveraging Microsoft Fluent UI for how things look.",
+  "scripts": {
+    "build": "rushx clean && concurrently --kill-others-on-fail \"tsc\" \"webpack --mode=production --env production\"",
+    "clean": "rimraf dist",
+    "package": "copyfiles -E -u 3 \"../Server/dist/**/*\" dist",
+    "start": "concurrently \"cd ../Server && rushx start:dev\" \"rushx start:app\"",
+    "start:app": "webpack-cli serve --mode=development --env development",
+    "test": "",
+    "prettier": "prettier --no-error-on-unmatched-pattern --write --config ../../.prettierrc --ignore-path=../../.prettierignore \"**/*.js\" \"**/*.jsx\" \"**/*.ts\" \"**/*.tsx\"",
+    "prettier:check": "prettier --no-error-on-unmatched-pattern --check --config ../../.prettierrc --ignore-path=../../.prettierignore \"**/*.js\" \"**/*.jsx\" \"**/*.ts\" \"**/*.tsx\"",
+    "lint": "eslint */**/*.{ts,tsx}",
+    "lint:fix": "eslint */**/*.{ts,tsx} --fix",
+    "lint:quiet": "eslint */**/*.{ts,tsx} --quiet"
+  },
+  "dependencies": {
+    "@azure/communication-chat": "1.1.0",
```

Comment:
You shouldn't need any of these. They are direct dependencies of `@azure/communication-react`. In fact, you don't want that so that you don't get versino desync.

Reply:
I took a try in the repo and got this:
![image](https://user-images.githubusercontent.com/11863655/140583407-3ef6d1a4-d149-4dea-8304-920204ad963a.png)

And this is the dependency folder of the project
![image](https://user-images.githubusercontent.com/11863655/140583521-361f434d-e1b6-49bb-8429-0b2e5273d693.png)

So it is actually under the subfolder of @azure/communication-react, but sometimes compiler isn't that smart to seek inside of subfolder

In general, we might just need to put all dependencies we directly use in the project into package.json - if the dependencies doesn't satisfy our @azure/communication-react, we should throw a warning like what Create React App does

BTW, in this project, version desync should be fine, which will be handled by rush itself